clio-util/src/main/scala/org/broadinstitute/clio/util/generic/CirceEquivalentCamelCaseLexer.scala

```diff
@@ -8,12 +8,12 @@ import s_mach.string.{LexResult, Lexer, RegexCharTransitionLexer}
   *
   * By default, s_mach only breaks camel-case through a single type of "boundary" detection:
   *
-  *   ([a-z])([A-Z]) -> $1_$2
+  *   ([a-z])([A-Z]) -> \$1_\$2
```

Comment:
Did enclosing it in a code block (`{{{ ... }}}`) not work? That has the added benefit of better formatting.